### ISSUE TITLE
split error in #args

### ISSUE THREAD

**Issue by trans** [defunkt/tomdoc#9]
tomdoc.rb:69:in `args': private method`split' called for nil:NilClass (NoMethodError)

```
require 'tomdoc/tomdoc'
tomdoc = TomDoc::TomDoc.new(comment)
tomdoc.args
```

**Comment by trans** [comment]
Probable fix:

```
    def args
      args = []
      last_indent = nil

      return args unless sections[1]

      ...
```

**Comment by trans** [comment]
Fixed in latest commit (e3a7f50a6e9b98beb41bf9303246b7e93a8bcf9d).
